File: main/src/main/java/net/fwitz/math/main/binary/complex/functions/trig/inverse/AcosZ.java
package net.fwitz.math.main.binary.complex.functions.trig.inverse;

import net.fwitz.math.binary.complex.Complex;
import net.fwitz.math.plot.binary.complex.ComplexFunctionPlot;

public class AcosZ {
    public static void main(String[] args) {
        new ComplexFunctionPlot("acos(z)")
                .computeFn(Complex::acos)
                .render();
    }
}

File: main/src/main/kotlin/net/fwitz/math/main/binary/complex/functions/trig/inverse/AcosZ.kt
package net.fwitz.math.main.binary.complex.functions.trig.inverse

import net.fwitz.math.binary.complex.Complex
import net.fwitz.math.plot.binary.complex.ComplexFunctionPlot

object AcosZ {
    @JvmStatic
    fun main(args: Array<String>) = ComplexFunctionPlot("acos(z)")
        .computeFn(Complex::acos)
        .render()
}